uaclient/config.py

```diff
@@ -793,6 +793,7 @@ def parse_config(config_path=None):
     cfg.update(env_keys)
     cfg["log_level"] = cfg["log_level"].upper()
     cfg["data_dir"] = os.path.expanduser(cfg["data_dir"])
+    cfg["config_path"] = config_path
```

Comment:
putting the config_path in here feels a little weird. If we have time, could we refactor slightly so config path is determined up in __init__ and set as an attribute on the UAConfig instance rather than in the cfg dict? Just an idea, feel free to disagree

Reply:
I don't think it is that weird since the `config_path` can in fact be configured from the outside. However, I can do the refactoring if needed. @blackboxsw do you share @orndorffgrant feeling that this variable should not be include in the `config` dict ?

Reply:
True that it can be configured from the outside. The slight difference is that the rest of the contents of `cfg` are configurable via uaclient.conf, while `config_path` isn't.

I agree that it is not that weird though, lets not hold up this PR on this.